src/admin/components/JsonData/index.js

```diff
@@ -5,7 +5,7 @@ import './style.scss';
 
 export default class JsonData extends React.Component {
   static propTypes = {
-    data: PropTypes.object.isRequired,
+    data: PropTypes.shape({}).isRequired,
```

Comment:
I think I'd prefer that we change `react/forbid-prop-types` to allow `object` rather than use `shape({})` all over. Do we expect that we'll actually define what the shape is?